### ISSUE TITLE
Cannot get PSU status from commandline on a Mellanox SN2010 switch

### ISSUE THREAD

**Issue by donlon** [sonic-net/sonic-buildimage#14416]
#### Description

On a SN2010 switch that installed the latest SONiC build, `show platform psustatus` command is unable to get all the status information (like the model name and power dissipation) of the PSU.

#### Steps to reproduce the issue:
1. run `show platform psustatus` command

#### Describe the results you received:
```
$ show platform psustatus
PSU    Model    Serial    HW Rev    Voltage (V)    Current (A)    Power (W)    Status    LED
-----  -------  --------  --------  -------------  -------------  -----------  --------  -----
PSU 1  N/A      N/A       N/A       N/A            N/A            N/A          NOT OK    red
PSU 2  N/A      N/A       N/A       N/A            N/A            N/A          OK        green
```

#### Output of `show version`:

```
SONiC Software Version: SONiC.master.239158-20f1ab820
Distribution: Debian 11.6
Kernel: 5.10.0-18-2-amd64
Build commit: 20f1ab820
Build date: Thu Mar 23 12:56:51 UTC 2023
Built by: AzDevOps@vmss-soni000QAV

Platform: x86_64-mlnx_msn2010-r0
HwSKU: ACS-MSN2010
ASIC: mellanox
ASIC Count: 1
Serial Number: ...
Model Number: MSN2010-CB2F
Hardware Revision: A3

...
```

**Comment by keboliu** [comment]
As explained above, this is not an issue, I close this one.

**Comment by keboliu** [comment]
> Hi,
> 
> Thanks for your reply! I tried to run a similar command on Onyx OS on the same device, but it at least returns PSU information like power dissipation and capacity.
> 
> ```
> mlx-sn2010 [standalone: master] > show power
> -----------------------------------------------------------------------------------------
> Module  Device            Sensor  Power   Voltage  Current  Capacity  Feed  Status
>                                   [Watts] [Volts]  [Amp]    [Watts]
> -----------------------------------------------------------------------------------------
> MGMT    PS1               PS1 12V 34.03   12.16    2.80     460.00    AC    OK
> MGMT    PS2               -       -       -        -        -         -     FAIL
> 
> Total power used : 34.03 Watts
> Total power capacity : 460.00 Watts
> Total power available : 425.97 Watts
> Maximum consumed power of all turned on modules: 250.00 Watts
> ```
> 
> The OS version is
> 
> ```
> mlx-sn2010 [standalone: master] > show version
> Product name:      Onyx
> Product release:   3.10.4206
> ...
> ```
> 
> I guess the SONiC OS failed to retrieve these information from the hardware.

SONiC is designed to get all the PSU related info from some sysfs, for MSN2010, it doesn't expose that info via sysfs as other Mellanox platforms due to different HW/SW design. ONYX using some other ways to get them but that is not applicable to SONiC.
